go.mod

```diff
@@ -239,3 +239,4 @@ require (
 
 // Use custom version that supports configuration as flags until it lands upstream.
 replace github.com/hashicorp/terraform-plugin-docs v0.5.1 => github.com/jacobbednarz/terraform-plugin-docs v0.5.1-0.20211203000021-49f8b60f2070
+replace github.com/cloudflare/cloudflare-go v0.32.0 => github.com/aplunk/cloudflare-go v0.99.0
```

Comment:
This project handles dependency version bumps (including upstream changes from cloudflare-go) independently of the standard PR process using automation. This allows the dependency upgrades to land without causing merge conflicts in multiple branches and handled in a consistent way. The exception to this is security related dependency upgrades but they should be co-ordinated with the maintainer team privately.

Please remove the changes to the go.mod or go.sum files from this PR in order to proceed with review and merging.

Reply:
Fixed, thanks for catching!